### DISCUSSION TITLE
Why are relax and auto pilot incompatible with perfect and sudden death?

### DISCUSSION THREAD

**Discussion by hburn7** [ppy/osu#19198] [Q&A] [answered]
If I am going for a relax/auto pilot SS or FC, it would be nice to be able to use the perfect or sudden death mods to ensure I get the desired outcome. I don't understand why relax/AP would be incompatible with these mods.

Edit: Also the tooltips are inconsistent for auto pilot.
Edit 2: Reflects that both Relax and AP are incompatible with PF/SD
![osu_2022-07-17_14-44-12](https://user-images.githubusercontent.com/38370573/179420330-72f16f5b-e6d8-46ec-8cba-3fd8015e6d50.png)
![osu_2022-07-17_14-44-16](https://user-images.githubusercontent.com/38370573/179420331-c7abd468-7208-413f-bbbc-6606ba76da95.png)

**Comment by peppy** [comment, marked as answer]
you can't fail in relax
